service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/generated/GeneratedOrder.java

```diff
@@ -35,18 +39,39 @@
     private final String uploader;
     private final String uploadedOrderDescription;
     @JsonSerialize(contentConverter = BasicChildConverter.class)
-    private final List<Element<Child>> children;
+    private List<Element<Child>> children;
     private String removalReason;
 
     @JsonIgnore
     public boolean isRemovable() {
         GeneratedOrderTypeDescriptor descriptor = GeneratedOrderTypeDescriptor.fromType(this.type);
         return (descriptor.getType() == BLANK_ORDER)
```

Comment:
would it be easier to do what it is not equal to at this point 😄

Reply:
Yeah it probably would but I think being more explicit is better.

Reply:
If you leave it explicit, it will prevent for inadvertently adding a new type of order and inherently by default adding it to the remove tool (without having a proper analysis and just as an maybe unwanted side effect).